tools/zetasql_helper/zetasql_helper/local_service/local_service.cc

```diff
@@ -81,5 +83,35 @@ absl::Status ZetaSqlHelperLocalServiceImpl::GetAllKeywords(const GetAllKeywordsR
   return absl::Status();
 }
 
+
+absl::Status ZetaSqlHelperLocalServiceImpl::FixColumnNotGrouped(
+    const FixColumnNotGroupedRequest& request,
+    FixColumnNotGroupedResponse* response) {
+
+  std::string fix_query;
+  ZETASQL_RETURN_IF_ERROR(
+      ::bigquery::utils::zetasql_helper::FixColumnNotGrouped(
+          request.query(), request.missing_column(), request.line_number(), request.column_number(),
+          &fix_query
+      ));
+
+  response->set_fixed_query(std::string(fix_query));
```

Comment:
why need cast here? it is std::string right ?

Reply:
Done